doc/source/serve/model_composition.md

```diff
@@ -128,17 +128,20 @@ Example:
 :language: python
 ```
 
-## Advanced: Pass a DeploymentResponse "by reference" [DEPRECATED]
+## Advanced: Pass a DeploymentResponse in a nested object [DEPRECATED]
 
 :::{warning}
-Passing a `DeploymentResponse` to downstream handle calls by reference, i.e. in nested objects, is deprecated and will be removed in the next release.
-Please use `DeploymentResponse._to_object_ref()` instead to pass the corresponding object reference in nested objects.
+Passing a `DeploymentResponse` to downstream handle calls in nested objects is deprecated and will be removed in the next release.
```

Comment:
We should mention what still works (the non nested way). Is there documentation on that outside this section?

Reply:
I've mentioned in the bottom section of this warning that passing it as a top level arg is still supported. This functionality itself is documented in the main section of this page.